src/js/components/PageHeader.js

```diff
@@ -59,22 +60,25 @@ class PageHeader extends React.Component {
 
 PageHeader.defaultProps = {
   className: 'container container-fluid container-pod container-pod-short flush flush-top',
-  mediaWrapperClassName: 'media-object-spacing-wrapper media-object-spacing-narrow media-object-offset',
-  dividerClassName: 'container-pod container-pod-short flush-top container-pod-divider-bottom container-pod-divider-bottom-align-right container-pod-divider-inverse',
-  titleClassName: 'flush inverse',
+  dividerClassName: 'container-pod container-pod-short flush-top flush-bottom container-pod-divider-bottom container-pod-divider-bottom-align-right container-pod-divider-inverse',
   iconClassName: 'icon icon-large icon-image-container icon-app-container',
+  mediaWrapperClassName: 'media-object-spacing-wrapper media-object-spacing-narrow media-object-offset',
+  subTitleClassName: 'emphasize',
+  titleClassName: 'flush inverse'
 };
 
 PageHeader.propTypes = {
   icon: React.PropTypes.node,
+  navigationTabs: React.PropTypes.node,
   subTitle: React.PropTypes.node,
   title: React.PropTypes.string,
 
   className: React.PropTypes.string,
   dividerClassName: React.PropTypes.string,
+  iconClassName: React.PropTypes.string,
   mediaWrapperClassName: React.PropTypes.string,
-  titleClassName: React.PropTypes.string,
-  iconClassName: React.PropTypes.string
+  subTitleClassName: React.PropTypes.string,
+  titleClassName: React.PropTypes.string
```

Comment:
Sorry for being vague! I was referring to the ABC ordering of the prop keys defined in this Object :) Just a nitpick - looks good!